src/styles/index.ts

```diff
@@ -3207,6 +3207,20 @@ const styles = (theme: ThemeColors) =>
             marginLeft: 3,
         },
 
+        WorkspaceRightColumn: {
+            marginLeft: 99,
```

Comment:
Hmm why are we taking this kind of approach with hard-coded values instead of just using flex to fill up space?

Reply:
Where did you get 99 from?

Reply:
Hmm but you can tell that the columns in the header don't perfectly match up with the content below. Owner looks like it's too far to the left for instance, compared to the content below it. 

Why wouldn't that invisible 4th column header just match the width of the 4th column in the table?

Reply:
Right, but if there is no badge present, that means we just have a bunch of dead space?